example/apu2_gpiod.c

```diff
@@ -0,0 +1,236 @@
+/* **********************************************************************
+**
+** Copyright (c) 2021 Raimo Niskanen <raimo (at) niskanen (dot) nu>
+**
+** Permission to use, copy, modify, and distribute this software for any
+** purpose with or without fee is hereby granted, provided that the above
+** copyright notice and this permission notice appear in all copies.
+**
+** THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL WARRANTIES
+** WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED WARRANTIES OF
+** MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR
+** ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES OR ANY DAMAGES
+** WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS, WHETHER IN AN
+** ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION, ARISING OUT OF
+** OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS SOFTWARE.
+**
+** **********************************************************************
+**
+** apu2_gpiod - a daemon to control the APU2/3 LEDs on OpenBSD
+**
+** WARNING: Do not use this code on any other hardware than
+** PC Engines APU2 (tested) or APU3 (should work).
+** The direct memory accesses may do anything including damage
+** to other hardware.
+**
+** See apu_gpio_lib from PC Engines.  This program is built
+** as one of its examples, and can be copied to for example
+** /root/bin/.
+**
+** Start the daemon from rc.securelevel while direct memory
+** access is allowed.
+**
+** Example rc.securelevel line:
+**     /root/bin/apu2_gpiod /var/apu2_gpio
+**
+** The FIFO /var/apu2_gpio has to be created:
+** # mkfifo -m 600 /var/apu2_gpio
+**
+** Then, after reboot, the LEDS can be controlled by writing
+** sequences of 1-character LED commands to the FIFO.
+** The LEDs are set to ON,OFF,OFF from left to right
+** when the daemon starts.
+**
+** LED commands:
+**   A,B,C  Set left,middle,right LED to ON
+**   a,b,c  Set left,middle,right LED to OFF
+**   1,2,3  Toggle left,middle,right LED
+**   All other characters are ignored
+**
+** Examples:
+** # echo abC > /var/apu/gpio	# Set only the right LED to ON 
+** # echo 2: > /var/apu/gpio	# Toggle the middle LED
```

Comment:
For me works:
echo abC > /var/apu2_gpio
echo 2: > /var/apu2_gpio

Reply:
@RaimoNiskanen Is it something that should be adjusted?

Otherwise, I'm glad to merge this, and thank you again for this contribution!

Reply:
Sry; what would be adjusted?

Reply:
Oh, now I see.  I will have a look...

Reply:
Yes.  I have an inconsistency in the documenting comments.  I will fix it!

Reply:
Fixed in [1ef622d](https://github.com/pcengines/apu_gpio_lib/pull/4/commits/1ef622df51e993b9c2d5f892136f2951c18d3e45)

Reply:
@Pokisiekk Please confirm, so we can close it.

Reply:
I confirm